pkg/content_manager.go

```diff
@@ -46,7 +47,7 @@ func (cm *ContentManager) Download(localFile string, opts PackageCreateOpts) err
 	return nil
 }
 
-func (cm *ContentManager) Sync(dir string, clean bool, packageOpts PackageCreateOpts) error {
+func (cm *ContentManager) SyncDir(dir string, clean bool, replace bool, packageOpts PackageCreateOpts) error {
```

Comment:
how about removing this clean bool from here ; and from outside just call additionally CleanDir ?